src/UniversalRewardsDistributor.sol

```diff
@@ -4,75 +4,81 @@ pragma solidity 0.8.21;
 import {PendingRoot, IUniversalRewardsDistributor} from "./interfaces/IUniversalRewardsDistributor.sol";
 
 import {ErrorsLib} from "./libraries/ErrorsLib.sol";
+import {EventsLib} from "./libraries/EventsLib.sol";
 import {SafeTransferLib, ERC20} from "@solmate/utils/SafeTransferLib.sol";
 
 import {MerkleProof} from "@openzeppelin/utils/cryptography/MerkleProof.sol";
-import {EventsLib} from "./libraries/EventsLib.sol";
 
 /// @title UniversalRewardsDistributor
 /// @author Morpho Labs
 /// @custom:contact security@morpho.org
-/// @notice This contract enables the distribution of various reward tokens to multiple accounts using different permissionless Merkle trees.
-///         It is largely inspired by Morpho's current rewards distributor:
-///         https://github.com/morpho-dao/morpho-v1/blob/main/src/common/rewards-distribution/RewardsDistributor.sol
+/// @notice This contract enables the distribution of various reward tokens to multiple accounts using different
+/// permissionless Merkle trees. It is largely inspired by Morpho's current rewards distributor:
+/// https://github.com/morpho-dao/morpho-v1/blob/main/src/common/rewards-distribution/RewardsDistributor.sol
 contract UniversalRewardsDistributor is IUniversalRewardsDistributor {
     using SafeTransferLib for ERC20;
 
-    /// @notice The merkle tree's roots of a given distribution.
+    /* STORAGE */
+
+    /// @notice The merkle tree's root of a given distribution.
     bytes32 public root;
 
     /// @notice The optional ipfs hash containing metadata about the root (e.g. the merkle tree itself).
     bytes32 public ipfsHash;
 
     /// @notice The `amount` of `reward` token already claimed by `account` for one given distribution.
-    mapping(address => mapping(address => uint256)) public claimed;
+    mapping(address account => mapping(address reward => uint256 amount)) public claimed;
```

Comment:
@Rubilmax removed it [here](https://github.com/morpho-org/universal-rewards-distributor/pull/35), I don't have strong opinion on it.

Reply:
Because we do reference on it in the @notice I feel that is good to write them. Else the "`" are useless.